chromium/background-scripts/update.js

```diff
@@ -31,7 +31,7 @@ let store,
   background_callback;
 
 // how often we should check for new rulesets
-const periodicity = 10;
+const periodicity = 7200;
```

Comment:
@Hainish isn't this just 2 hours instead of 2 days in seconds?

Reply:
You're definitely right!  Thank you.

Reply:
It's those little bugs which are killers.  Glad you caught that.

Reply:
033f82d looks good to me, thanks! 😄